websites/B/BoxSeries/presence.ts

```diff
@@ -0,0 +1,162 @@
+const presence = new Presence({
+		clientId: "955350055034945576"
+	}),
+	strings = presence.getStrings({
+		play: "general.playing",
+		pause: "general.paused",
+		browsing: "general.browsing"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video = {
+	current: 0,
+	duration: 0,
+	paused: true
+};
+
+presence.on(
+	"iFrameData",
+	(data: { current: number; duration: number; paused: boolean }) => {
+		video = data;
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const [logo, cover, privacy, buttons, timestamps] = await Promise.all([
+			presence.getSetting<number>("logo"),
+			presence.getSetting<boolean>("cover"),
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("buttons"),
+			presence.getSetting<boolean>("timestamps")
+		]),
+		titleMain =
+			document.querySelector("#dt_contenedor > div.module > div > div > h1")
+				?.textContent ?? "pesquisa não encontrada",
+		titleEp =
+			document.querySelector("#single > h1.epih1")?.textContent.split(" - ") ??
+			"pesquisa não encontrada",
+		pathArray = document.location.toString().split("/"),
+		presenceData: PresenceData = {
+			largeImageKey: ["default", "default_horizontal_tp"][logo],
+			startTimestamp: browsingTimestamp
+		};
+	if (new URLSearchParams(location.search).has("s")) {
+		presenceData.details = "Pesquisando po";
+		presenceData.state = `${
+			document.querySelector(
+				"#dt_contenedor > div.module > div > div > div > header > h1 > span"
+			)?.textContent ?? "pesquisa não encontrada"
+		}`;
+		presenceData.smallImageKey = "search";
+	} else {
+		switch (pathArray[3]) {
+			case "calendario":
+				presenceData.details = "Vendo Calendário do Dia";
+				presenceData.smallImageKey = "reading";
+				break;
+			case "generos":
+				presenceData.details = "Vendo Gêneros";
+				if (!privacy && pathArray[4]) presenceData.state = titleMain;
+				presenceData.smallImageKey = "reading";
+				break;
+			case "anos":
+				presenceData.details = "Vendo anos";
+				presenceData.smallImageKey = "reading";
+				break;
+			case "release":
+				presenceData.details = "Vendo Liberação";
```

Comment:
Não sei até que ponto é que isto é correto no brasil porque soa estranho mas pronto
```suggestion
				presenceData.details = "Vendo liberação";
```